### DISCUSSION TITLE
DA1469x: Bus Resets on HID Generic InOut Port

### DISCUSSION THREAD

**Discussion by KWolfe81** [hathach/tinyusb#886] [Q&A]
I'm experimenting with TinyUSB support on the DA1469x and am running on both the da1469x_dk_pro & on a custom PCB.  My project is pretty much just the hid_generic_inout example.  If I connect to the device within the first few seconds, I can establish a connection and everything his hunky-dory.  

If, after enumeration, I don't attempt to communicate, I observe a USB Bus Reset signal along with a Suspend.  The Suspend is expected, but not the Reset.  In any case, I lose enumeration and cannot reconnect short of resetting the device.

Any tips on where to look or can I provide other information which might assist in getting to bottom of this? 

Thanks!

**Comment by hathach** [comment]
It is probably be around the bus signal detection. I haven't tested much with this chip. Though suspended can be mistaken as disconnected event (they are similar in term of D+/D- state) and only different in VBUS (suspend still has vbus). 
https://github.com/hathach/tinyusb/blob/master/src/portable/dialog/da146xx/dcd_da146xx.c#L705
